File: lec002/numlen.java
import java.util.Scanner;
public class numlen{
    public static Scanner scn=new Scanner(System.in);
    public static void main(String[] args){
        int n,count=0,k;
        System.out.print("Enter number: ");
        n=scn.nextInt();
        System.out.println();
        while(n>10){
            k=n%10;
            n=n/10;
            count++;
        }
        System.out.println(count+1);
    }
    
}

File: lec002/numlen.cpp
#include<iostream>
using namespace std;
int main(int args,char** argv){
    cout<<"Enter num: ";
    int n, count=0;
    cin>>n;
    while (n>10){
        int k=n%10;
        n=n/10;
        count++;
    }
    cout<<count+1;
    return 0;
}
